build-tools/plots-to-appinsights/AppInsights.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using System;
+using System.Collections.Generic;
+
+namespace ProcessPlotCSVFile
```

Comment:
Namespace should have a `Xamarin.Android.Tools` prefix, e.g. `Xamarin.Android.Tools.Pots`.

Reply:
Ah, good to know for any future tools.
Commit: 8cf10948bf271e7e37d026afd255c6003ad24cbd